### PR TITLE
refactor: use embedded resources

### PR DESCRIPTION
**Pull request by zregvart**
The way we bundle resources within the executable is problematic.
Resources need to be included by running code generation, so if skipped
older version of the resource will be included; and the generated .go
files contain the resources as byte arrays in hex, nullifying the
utility of diff tools and making resolving conflicts difficult.

This instead uses the `//go:embed` directive to embed the resources at
compilation time, making sure that the most recent version of the
resource is included and making potential merge conflicts explicit and
simpler to resolve.

### CODE REVIEW

**Comment by mkralik3** [comment]
retest this please
(QE tests fixed)

File: install/operator/pkg/generator/generator.go [hunk 1]
```diff
@@ -186,7 +171,7 @@ func Render(filePath string, context interface{}) ([]unstructured.Unstructured,
 		// It's ok if a template chooses not to generate any resources..
 
 	default:
-		return nil, fmt.Errorf("Unexptected yaml unmarshal type: %v", obj)
+		return nil, fmt.Errorf("unexptected yaml unmarshal type: %v", obj)
```
Comment (phantomjinx) [inline install/operator/pkg/generator/generator.go]: erroneous "t" in the middle of "unexpected"

Comment (phantomjinx) [inline install/operator/pkg/generator/generator.go]: erroneous "t" in the middle of "unexpected"

**Review by phantomjinx — APPROVED** [review]

File: install/operator/pkg/generator/generator.go (hunk 1 above)
Comment (zregvart) [reply install/operator/pkg/generator/generator.go]: Thanks!

**Pull request merged**